packages/devextreme/js/__internal/scheduler/m_utils_time_zone.ts

```diff
@@ -130,10 +225,11 @@ const hasDSTInLocalTimeZone = () => {
   return startDate.getTimezoneOffset() !== endDate.getTimezoneOffset();
 };
 
-const isEqualLocalTimeZoneByDeclaration = (timeZoneName, date) => {
+const getOffset = (date) => -date.getTimezoneOffset() / MINUTES_IN_HOUR;
+
+const isEqualLocalTimeZoneByDeclarationOld = (timeZoneName: string, date: Date): boolean => {
   const year = date.getFullYear();
-  const getOffset = (date) => -date.getTimezoneOffset() / 60;
-  const getDateAndMoveHourBack = (dateStamp) => new Date(dateStamp - 3600000);
+  const getDateAndMoveHourBack = (dateStamp) => new Date(dateStamp - MS_IN_HOUR);
```

Comment:
Minor

It had been here before, but let's move out the inner scope function to outer scope
E.g.
```typescript
const getDateAndMoveHourBack = (dateStamp) => new Date(dateStamp - toMs('hour'));

const isEqualLocalTimeZoneByDeclarationOld = (...) => { ... } 
```